NGitLab/Models/FileArchiveFormat.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Runtime.Serialization;
+
+namespace NGitLab.Models;
+
+public enum FileArchiveFormat
+{
+    [EnumMember(Value = ".bz2")]
```

Comment:
I've used `EnumMemberAttribute` here to map the enum value to the required file extension. I couldn't find anywhere else in the project where you were doing something like this. 

Previously in other projects I have used `System.ComponentModel.DescriptionAttribute` for this kind of thing, but if there is a way you'd prefer me to implement this here, let me know and I'll refactor to suit.

Reply:
Using `EnumMemberAttribute` is the way to go.